### DISCUSSION TITLE
Please check

### DISCUSSION THREAD

**Discussion by luorixiangyang** [minio/minio#12241] [Q&A] [answered]
We found that in the MINIO distribution solution, the service will spend twice as much storage space. The data objects will be separated to encode the data and stored in differents data nodes.

**So I have a question**: Is there a solution that does not require 2 times the space for storage, but uses a 1:1 storage scheme, so we may miss the backup and rescue data functions, but the current storage resources are insufficient The reason, I had to use a distributed solution to store more than 200 TB of data.

thanks. let me know if you can take help.

**Comment by harshavardhana** [comment, marked as answer]
MinIO uses erasure coding so there is no way to provide protection while keeping 1:1 - you need some form of parity. You can control the parity using STORAGE_CLASS configuration. 

Please look at https://min.io/product/erasure-code-calculator the calculator to understand the expected ratio efficiency.
